src/Codeception/Util/ActionSequence.php

```diff
@@ -12,55 +22,57 @@
  * ActionSequence::build()->click('do')->click('undo');
  * ```
  *
- * @method $this see([optional])
- * @method $this dontSee([optional])
- * @method $this seeElement([optional])
- * @method $this dontSeeElement([optional])
- * @method $this click([optional])
- * @method $this wait([optional])
- * @method $this waitForElementChange([optional])
- * @method $this waitForElement([optional])
- * @method $this waitForElementVisible([optional])
- * @method $this waitForElementNotVisible([optional])
- * @method $this waitForText([optional])
- * @method $this submitForm([optional])
- * @method $this seeLink([optional])
- * @method $this dontSeeLink([optional])
- * @method $this seeCheckboxIsChecked([optional])
- * @method $this dontSeeCheckboxIsChecked([optional])
- * @method $this seeInField([optional])
- * @method $this dontSeeInField([optional])
- * @method $this seeInFormFields([optional])
- * @method $this dontSeeInFormFields([optional])
- * @method $this selectOption([optional])
- * @method $this checkOption([optional])
- * @method $this uncheckOption([optional])
- * @method $this fillField([optional])
- * @method $this attachFile([optional])
- * @method $this seeNumberOfElements([optional])
- * @method $this seeOptionIsSelected([optional])
- * @method $this dontSeeOptionIsSelected([optional])
+ * @method $this see[optional]
+ * @method $this dontSee[optional]
+ * @method $this seeElement[optional]
+ * @method $this dontSeeElement[optional]
+ * @method $this click[optional]
+ * @method $this wait[optional]
+ * @method $this waitForElementChange[optional]
+ * @method $this waitForElement[optional]
+ * @method $this waitForElementVisible[optional]
+ * @method $this waitForElementNotVisible[optional]
+ * @method $this waitForText[optional]
+ * @method $this submitForm[optional]
+ * @method $this seeLink[optional]
```

Comment:
What's the meaning of `[optional]` here?

Reply:
Sincerely I don't know. I just know that as it was before `([optional])` it was wrong.

Reply:
It was a really really **really** old way of denoting that one method could have one or more optional parameters without actually specifying them. Perhaps it is best to include the current API, although this may become out of date over time.